CHANGELOG.md

```diff
@@ -1,5 +1,49 @@
 # Changelog
 
+### 0.1.0
+
+Features
+
+* Add IngressLink support (#58) @Dean-Coakley
+* Add AppProtect User Defined Signatures support (#55) @Dean-Coakley
+* Graduate Policies. Add enablePreviewPolicy flag support. (#56) @Dean-Coakley
+* Update secret type of default secret to TLS (#39) @pleshakov
+
+Bug Fixes
+
+* Fix replicas and service to be optional fields (#71) @Dean-Coakley
+* Make enableCRDs optional (#70) @Dean-Coakley
+* Fix Service to be an optional field. Add support for updating ExtraLabels. (#66) @Dean-Coakley
+* Fix SCC resource to only affect KIC pods (#65) @Dean-Coakley
+
+Documentation
+
+* Update IC compatibility in changelog (#54) @pleshakov
+
+Maintenance
```

Comment:
there is no need to put things into the changelog  that don't affect our users: they care about features, bug fixes, changes, docs, but not about dependencies or chore PRs. 

I know we have that in the auto generated changelog, but it's just there for us as a reference of all PRs, so that we don't loose anything when writing a changelog.

could you remove this whole  group?